release-notes/PA/PA_1.3.1.md

```diff
@@ -0,0 +1,108 @@
+---
+uid: PA_1.3.1
+---
+
+# PA 1.3.1
+
+> [!NOTE]
+> This version requires that **SRM 1.2.28 [CU1]** is installed.
+
+## New features
+
+#### Assigning tasks to a specific user or group [ID_35082]
+
+While up to now Process Automation created user tasks without assigning them to specific users, now you can configure a process definition so that tasks are assigned or reassigned to a user or a group of users when they are created.
+
+During the configuration of a process definition,you can select a Process DOM field for either the group or the user so that the group or user name can be retrieved when the user task gets created. With the *Fixed* option you can already pre-select the exact group and user. However, if a Process DOM field is selected for a group, you cannot pre-select a fixed user. The *Any* option allows you to postpone the selection of the group or user to the time when the task gets created.
+
```

Comment:
Can you add a warning stating that this is first step towards being able to assign user tasks to specific users/groups? This is for now the configuration part; nothing will be done with the user/user group info.

Reply:
Done!